frontend/lib/src/components/elements/CodeBlock/StreamlitSyntaxHighlighter.tsx

```diff
@@ -34,7 +34,7 @@ export default function StreamlitSyntaxHighlighter({
   children,
 }: StreamlitSyntaxHighlighterProps): ReactElement {
   return (
-    <StyledCodeBlock>
+    <StyledCodeBlock className="stCodeBlock" isMarkdown={false}>
```

Comment:
should we be using data-testid in the e2e test?

Reply:
Hmm, in general yes. In that case, I added the `className` since the CodeBlock for Markdown also has the same class name. So, mostly for consistency. But I can also add a data test id.

Reply:
Added test id and used it in e2e tests.